test/extensions/flexijoins.jl

```diff
@@ -0,0 +1,22 @@
+import GeometryOps as GO, GeoInterface as GI
+using FlexiJoins, DataFrames
+
+pl = GI.Polygon([GI.LinearRing([(0, 0), (1, 0), (1, 1), (0, 0)])])
+pu = GI.Polygon([GI.LinearRing([(0, 0), (0, 1), (1, 1), (0, 0)])])
+poly_df = DataFrame(geometry = [pl, pu], color = [:red, :blue])
+
+points = tuple.(rand(100), rand(100))
+points_df = DataFrame(geometry = points)
+
+@testset "Basic integration" begin
+
+    @test_nowarn joined_df = FlexiJoins.innerjoin((poly_df, points_df), by_pred(:geometry, GO.contains, :geometry))
```

Comment:
It would be nice to wrap this and avoid the `:geometry` `:geometry` part

Like this maybe:
```julia
joined_df = GO.innerjoin(poly_df, points_df, GO.contains)
```

Reply:
I see what you're saying here now - but until we resolve the DataFrames/GI.geometrycolumn/ArchGDAL mess, it's probably best to keep this explicit IMO :D

Reply:
Is it actually a mess? It mostly just works

Reply:
In this case yes - GADM for example only outputs tables with `geom` columns,  as do many other ArchGDAL-based loaders.

Reply:
ArchGDAL tables are fine, GADM returns a NamedTuple. Theres an issue for that